src/components/views/dialogs/SessionRestoreErrorDialog.js

```diff
@@ -29,6 +29,12 @@ export default React.createClass({
         onFinished: React.PropTypes.func.isRequired,
     },
 
+    componentDidMount: function() {
+        if (this.refs.bugreportLink) {
+            this.refs.bugreportLink.focus();
```

Comment:
Should we not be auto-focussing the "Continue anyway" button instead?

Reply:
I like to focus the bug report link when it's available and focus the button when that is not there. As I think we should encourage people to report their issues when they find some. Focusing the button would mean screen reader users have to shift+tab in order to focus that link. I assume by default almost every keyboard user navigates by pressing the tab key not shift+tab. Still this is my feeling only and if you like me to change it then I will do it.

Reply:
Okay, fair reasoning. This dialog shouldn't come up with any frequency anyway.